crates/solidity/outputs/typescript/lib/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I also suggest removing the `.husky` folder with the pre-commit hook.

Reply:
done